integration/benchmark/follower.go

```diff
@@ -114,8 +114,21 @@ Loop:
 		default:
 		}
 
+		nextHeight := f.Height() + 1
```

Comment:
could you added a comment the txFollowerImpl struct explaining the threading model?  This code is only correct for single writer / multiple reader

Reply:
Seconding, in general we have a lack of safety related comments.